go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/formancehq/ledger
 
 go 1.22.1
 
-toolchain go1.22.7
+toolchain go1.23.2
```

Comment:
Is this upgrade intentional?

Reply:
hum, no.
I don't have go 1.23 locally, it's strange.